.pep8speaks.yml

```diff
@@ -1,2 +1,2 @@
 pycodestyle:
-    max-line-length: 100  # Default is 79 in PEP 8
```

Comment:
I'd be inclined to keep at 100: 120 is harder for multiple split screens on my computer.

Reply:
Sure - I'll push a refractor to satisfy pep8 soon